test/env.sh

```diff
@@ -2,12 +2,12 @@
 
 : "${SPLUNK_OPERATOR_IMAGE:=splunk/splunk-operator:latest}"
 : "${SPLUNK_ENTERPRISE_IMAGE:=splunk/splunk:latest}"
-: "${CLUSTER_PROVIDER:=kind}"
-: "${CLUSTER_NAME:=integration-test-cluster-eks}"
+: "${CLUSTER_PROVIDER:=eks}"
```

Comment:
Dont think so these changes need to be there!

Reply:
Oh no.. :-). removing it.